Tropos/Models/DailyForecast.swift

```diff
@@ -0,0 +1,15 @@
+import Foundation
+
+@objc class DailyForecast: NSObject {
+    let date: NSDate
+    let conditionsDescription: String
+    let highTemperature: Temperature
+    let lowTemperature: Temperature
+    
+    init(json: NSDictionary) {
```

Comment:
@jakecraige I will also change JSON type to `Dictionary<String, AnyObject>` to keep it more Swifty.